### PR TITLE
Fixes for mcl* compatibility.

### PR DESCRIPTION
**Pull request by MysticTempest**
Hi, just a quick fix to allow for the crafting bench to be craftable in mcl* games; as well as those with the 'default' mod.
And, fix not being able to properly dig by hand, the bench; in mcl* games.

### CODE REVIEW

File: init.lua [hunk 1]
```diff
@@ -32,6 +32,9 @@ local groups
 local mcl_hardness
 local mcl_blast_res
 local sounds
+local steel_ingot
+local wood
+local tree
```
Comment (Niklp09) [inline init.lua]:
```suggestion
local steel_ingot, wood, tree
```

File: init.lua [hunk 2]
```diff
@@ -76,9 +79,9 @@ elseif has_mcl then
 		'listring[current_player;main]'..
 		'listring[current_name;src]'..
 		'listring[current_player;main]'
-	groups = {axey=2, handy=2, flammable=-1, container = 4}
+	groups = {axey=2, handy=1, flammable=-1, container = 4}
```
Comment (Niklp09) [inline init.lua:77]: Why is this change required?

**Review by Niklp09 — COMMENTED** [review]
See my comments, LGTM overall.

File: init.lua (hunk 2 above)
Comment (MysticTempest) [reply init.lua:77]: "Handy = 2" doesn't exist in mcl games. "Handy = 1" is what's required to allow the node to drop when dug by hand; otherwise it's destroyed. Increasing the hardness value is how the dig time is increased.

Comment (Niklp09) [reply init.lua:77]: Ah sorry, i thought the `handy` group does exist in mtg too. Then this is fine-

File: init.lua (hunk 1 above)
Comment (MysticTempest) [reply init.lua]: Do you want the others in a row as well?

```local groups, mcl_hardness, mcl_blast_res, sounds```  
```local steel_ingot, wood, tree```

Comment (Niklp09) [reply init.lua]: Would be good IMO, but thats just style.

Comment (MysticTempest) [reply init.lua]: Okay, done.

**Review by Niklp09 — APPROVED** [review]
Will merge in a few days unless there are objections.
